File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT1001U01GetGroupKeyInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
	public class NuroOUT1001U01GetGroupKeyInfo
	{
		private String _groupKey;

		public String GroupKey
		{
			get { return this._groupKey; }
			set { this._groupKey = value; }
		}

		public NuroOUT1001U01GetGroupKeyInfo() { }

		public NuroOUT1001U01GetGroupKeyInfo(String groupKey)
		{
			this._groupKey = groupKey;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT1001U01GetGroupKeyInfo.java
package nta.med.data.model.ihis.nuro;

import java.io.Serializable;

public class NuroOUT1001U01GetGroupKeyInfo implements Serializable{
	private String groupKey;

	public NuroOUT1001U01GetGroupKeyInfo(String groupKey) {
		super();
		this.groupKey = groupKey;
	}

	public String getGroupKey() {
		return groupKey;
	}

	public void setGroupKey(String groupKey) {
	}
	
	
}
